src/engine/client/ClientApplication.cpp

```diff
@@ -126,7 +128,7 @@ class ClientApplication : public Application {
 
         void Shutdown(bool error, Str::StringRef message) override {
             #if defined(_WIN32) || defined(BUILD_GRAPHICAL_CLIENT)
-                if (error) {
+                if (error && client_errorPopup.Get()) {
```

Comment:
should we print to stdout instead?

Reply:
The code calling for `Shutdown()` always prints the message as a warning as far as I know, before actually calling `Shutdown()`.

Reply:
https://github.com/DaemonEngine/Daemon/blob/7d15e9236bb8fd40f027d1d8248fcb42f65f5e55/src/engine/framework/System.cpp#L551-L562